replications/internal/queue_management.go

```diff
@@ -439,15 +439,15 @@ func (qm *durableQueueManager) EnqueueData(replicationID platform.ID, data []byt
 	return nil
 }
 
-func (qm *durableQueueManager) newReplicationQueue(id platform.ID, orgID platform.ID, localBucketID platform.ID, queue *durablequeue.Queue, maxAge int64) *replicationQueue {
+func (qm *durableQueueManager) newReplicationQueue(id platform.ID, orgID platform.ID, localBucketID platform.ID, queue *durablequeue.Queue, maxAgeSeconds int64) *replicationQueue {
 	logger := qm.logger.With(zap.String("replication_id", id.String()))
 	done := make(chan struct{})
 	// check for max age minimum
 	var maxAgeTime time.Duration
-	if maxAge < 0 {
+	if maxAgeSeconds < 0 {
 		maxAgeTime = defaultMaxAge
 	} else {
-		maxAgeTime = time.Duration(maxAge)
+		maxAgeTime = time.Duration(time.Duration(maxAgeSeconds) * time.Second)
```

Comment:
You can remove the outer `time.Duration`, `time.Duration(maxAgeSeconds) * time.Second` will be properly typed.